modules/microshift-data-cleaning-full-cleanup.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * microshift_troubleshooting/microshift-cleanup-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-data-cleaning-full-cleanup_{context}"]
+= Cleaning all data and configuration
+
+Cleanup all the {microshift-short} data and configuration by running the `microshift-cleanup-data` script.
+
+When you run the script with the `--all` argument, you perform the following cleanup actions:
+
+* All {microshift-short} services are stopped and disabled
+* Deletes all {microshift-short} pods
+* Deletes all container image storage
+* Reset network configuration
+* Deletes the `/var/lib/microshift` data directory
+* Deletes OVN-K networking configuration
+
+.Prerequisites
+. You are logged into the {microshift-short} as an administrator with root-user access.
+. You have filed a support case.
+
+.Procedure
+
+. Cleanup all the {microshift-short} data and configuration by running the `microshift-cleanup-data` script with the `--all` argument, by entering the following command:
```

Comment:
Also same comment about removing the description of the command when you can see what's in the command directly below:
```suggestion
. Cleanup all the {microshift-short} data and configuration by entering the following command:
```